packages/rx_bloc_cli/lib/src/utils/file_cleanup.dart

```diff
@@ -0,0 +1,46 @@
+import 'dart:io';
+
+/// Helper class used for cleaning up any redundant files after generation phase
+class FileCleanup {
+  static final _dirsToRemove = [
+    'lib/pages/',
+    '.tmp/',
+    'setup_scripts/',
+    'android/environments/',
+  ];
+  static final _filesToRemove = [
+    'test/widget_test.dart',
+    'lib/main.dart',
+    'lib/app.dart',
+    'lib/flavors.dart',
+    'assets.tmp.zip',
+    'flavorizr.yaml',
+    '.idea/runConfigurations/main_dart.xml',
+  ];
+
+  static List<String> get _itemsToRemove => [
+        ..._filesToRemove,
+        ..._dirsToRemove,
+      ];
+
+  /// Remove any generated files that are not needed (such as files created
+  /// using the `flutter create` command or by-products of adding flavors)
+  static Future<void> postGenerationCleanup(
+    Directory outputDirectory, {
+    List<String>? filesToRemove,
+  }) async {
+    final _toRemove = List<String>.from(_itemsToRemove);
+    if (filesToRemove != null) _toRemove.addAll(filesToRemove);
+    final projectPath = '${outputDirectory.absolute.path}/';
+
+    for (var path in _toRemove) {
+      try {
+        if (path.endsWith('/')) {
+          await Directory('$projectPath$path').delete(recursive: true);
+        } else {
+          await File('$projectPath$path').delete();
+        }
+      } catch (_) {}
```

Comment:
Do we really want to suppress the errors here ?

Reply:
The try-catch was added as a way of silencing any deletion errors (in case of deleting non-existent files or similar). The files we are talking about are dev files or scripts used for the generation purposes or leftovers from that. 

Do we really want to display errors for those files during the (post) code generation process?